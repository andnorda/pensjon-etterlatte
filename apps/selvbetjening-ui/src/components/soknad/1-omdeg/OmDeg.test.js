import { render } from "@testing-library/react";

describe("OmDeg", () => {
    it("skal teste om deg", () => {
        /* const neste = jest.fn();
        Modal.setAppElement = () => null;

        const { container } = render(<OmDeg neste={neste} />);
        //expect(container).toMatchSnapshot();*/
    });
});
