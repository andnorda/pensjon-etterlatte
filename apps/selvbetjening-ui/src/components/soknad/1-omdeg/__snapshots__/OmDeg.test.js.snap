// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Om deg og avd√∏d Snapshot 1`] = `
<div>
  <h1
    class="center navds-heading navds-heading--medium"
  >
    omDeg.tittel
  </h1>
  <form>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe skjemagruppering"
    >
      <div
        id="bostedsadresseBekreftet"
      >
        <fieldset
          aria-invalid="false"
          class="skjemagruppe inputPanelGruppe inline"
        >
          <legend
            class="skjemagruppe__legend"
          >
            omDeg.bostedsadresseBekreftet
          </legend>
          <div
            class="inputPanelGruppe__inner"
          >
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="bostedsadresseBekreftet"
                required=""
                type="radio"
                value="Ja"
              />
              <span
                class="inputPanel__label"
              >
                radiobuttons.ja
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="bostedsadresseBekreftet"
                required=""
                type="radio"
                value="Nei"
              />
              <span
                class="inputPanel__label"
              >
                radiobuttons.nei
              </span>
            </label>
          </div>
          <div
            aria-live="polite"
          />
        </fieldset>
      </div>
      <fieldset
        aria-invalid="false"
        class="skjemagruppe"
      >
        <div
          class="navds-grid"
        >
          <div
            class="navds-grid__cell navds-grid__cell--xs-12 navds-grid__cell--md-3 kol"
          >
            <div
              class="skjemaelement"
            >
              <label
                class="skjemaelement__label"
                for="kontaktinfo.telefonnummer"
              >
                omDeg.kontaktinfo.telefonnummer
              </label>
              <input
                aria-invalid="false"
                class="skjemaelement__input input--s"
                id="kontaktinfo.telefonnummer"
                placeholder="omDeg.kontaktinfo.telefonnummerPlaceholder"
                required=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="navds-grid__cell navds-grid__cell--xs-12 navds-grid__cell--md-6 kol"
          >
            <div>
              <div
                class="skjemaelement"
              >
                <label
                  class="skjemaelement__label"
                  for="kontaktinfo.epost"
                >
                  omDeg.kontaktinfo.epost
                </label>
                <input
                  aria-invalid="false"
                  class="skjemaelement__input input--fullbredde"
                  id="kontaktinfo.epost"
                  required=""
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-live="polite"
        />
      </fieldset>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe skjemagruppering"
    >
      <div
        id="oppholderSegINorge"
      >
        <fieldset
          aria-invalid="false"
          class="skjemagruppe inputPanelGruppe inline"
        >
          <legend
            class="skjemagruppe__legend"
          >
            omDeg.oppholderSegINorge
          </legend>
          <div
            class="skjemagruppe__description"
          >
            <div
              class="hvorforPanel"
            >
              <button
                aria-expanded="false"
                class="hvorforPanel__toggle"
                data-testid="hvorforPanel_toggle"
                type="button"
              >
                <span>
                  hvorforSpoerVi
                </span>
                <span>
                  <svg
                    fill="none"
                    height="1em"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M12 17L3 8.429 4.5 7l7.5 7.143L19.5 7 21 8.429 12 17z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
          <div
            class="inputPanelGruppe__inner"
          >
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="oppholderSegINorge"
                required=""
                type="radio"
                value="Ja"
              />
              <span
                class="inputPanel__label"
              >
                radiobuttons.ja
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="oppholderSegINorge"
                required=""
                type="radio"
                value="Nei"
              />
              <span
                class="inputPanel__label"
              >
                radiobuttons.nei
              </span>
            </label>
          </div>
          <div
            aria-live="polite"
          />
        </fieldset>
      </div>
      <div
        aria-live="polite"
      />
    </fieldset>
    <br />
    <fieldset
      aria-invalid="false"
      class="skjemagruppe navigasjon-footer"
    >
      <div
        class="navigasjon-rad"
      >
        <button
          class="navds-button navds-button--primary navds-button--medium"
          type="button"
        >
          <span
            class="navds-button__inner navds-body-short"
          >
            knapp.neste
          </span>
        </button>
      </div>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe navigasjon-rad"
    >
      <button
        class="navds-button navds-button--secondary navds-button--medium"
        id="avbryt-btn"
        type="button"
      >
        <span
          class="navds-button__inner navds-body-short"
        >
          knapp.avbryt
        </span>
      </button>
      <div
        aria-live="polite"
      />
    </fieldset>
  </form>
</div>
`;
